backend/src/test/java/gov/cdc/usds/simplereport/api/model/TestEventExportIntegrationTest.java

```diff
@@ -156,6 +156,107 @@ void testEventSerialization() throws Exception {
 
   @Test
   void testCorrectedEventSerialization() throws Exception {
+    TestEvent correctedTestEvent = _dataFactory.createTestEventCorrected(testEvent);
+    TestEventExport correctedTestEventExport = new TestEventExport(correctedTestEvent);
+
+    String actualStr = objectMapper.writeValueAsString(correctedTestEventExport);
+    JSONAssert.assertEquals(
+        "{"
+            + "\"Patient_last_name\":\"Astaire\","
+            + "\"Patient_first_name\":\"Fred\","
+            + "\"Patient_middle_name\":\"M\","
+            + "\"Patient_suffix\":null,"
+            + "\"Patient_race\":\"2106-3\","
+            + "\"Patient_DOB\":\"18990510\","
+            + "\"Patient_gender\":\"M\","
+            + "\"Patient_ethnicity\":\"N\","
+            + "\"Patient_street\":\"736 Jackson PI NW\","
+            + "\"Patient_street_2\":\"APT. 123\","
+            + "\"Patient_city\":\"Washington\","
+            + "\"Patient_county\":\"Washington\","
+            + "\"Patient_state\":\"DC\","
+            + "\"Patient_zip_code\":\"20503\","
+            + "\"Patient_country\":\"USA\","
+            + "\"Patient_phone_number\":\"202-123-4567\","
+            + "\"Patient_email\":\"fred@astaire.com\","
+            + "\"Patient_ID\":"
+            + correctedTestEventExport.getPatientId()
+            + ","
+            + "\"Patient_role\":\"RESIDENT\","
+            + "\"Patient_tribal_affiliation\":\"\","
+            + "\"Patient_preferred_language\":\"English\","
+            + "\"Employed_in_healthcare\":\"N\","
+            + "\"Resident_congregate_setting\":\"N\","
+            + "\"Result_ID\":"
+            + correctedTestEventExport.getResultID()
+            + ","
+            + "\"Corrected_result_ID\":\""
+            + testEvent.getInternalId().toString()
+            + "\","
+            + "\"Test_correction_reason\":\"Cold feet\","
+            + "\"Order_result_status\":\"C\","
+            + "\"Observation_result_status\":\"C\","
```

Comment:
Heads up for the reviewers: these are the relevant bits for the purposes of this PR. For reference:

<img width="473" alt="Screen Shot 2022-03-29 at 2 05 16 PM" src="https://user-images.githubusercontent.com/27730981/161859978-80154e95-3db7-408c-8b93-7760eb1a794d.png">

(here, `Result status` === `Order_result_status`)